aten/src/ATen/native/native_functions.yaml

```diff
@@ -13138,7 +13138,9 @@
   autogen: _native_multi_head_attention.out
 
 - func: _scaled_dot_product_attention(Tensor query, Tensor key, Tensor value, Tensor? attn_mask=None, float dropout_p=0.0, bool need_attn_weights=True, bool is_causal=False) -> (Tensor, Tensor)
+  python_module: nn
   variants: function
+  autogen: _scaled_dot_product_attention.out
```

Comment:
Why do we need this?

Reply:
we don't but it was new thing added to help with functionalization and the readme says if you can, add it to all new native functions.